arithmaticOperation.js

```diff
@@ -0,0 +1,23 @@
+/**
+*   Calculate the area of a rectangle.
+*
+*   length: The length of the rectangle.
+*   width: The width of the rectangle.
+*   
+*	Return a number denoting the rectangle's area.
+**/
+function getArea(length, width) {
```

Comment:
change the argument because one can pass length , width as string , undefined anything , the code will break. 

this should be : 

function getArea(length : number, width : number)

Reply:
We can define the type of parameter in .ts file only

Reply:
you can use .ts instead of .js 
If you still don't want to change , add checks in the existing function body , so that if I pass inappropriate parameter in the function , this won't break the code.